### PR TITLE
chore: upgrade msfssdk to 0.4.0, remove fork

### PR DESCRIPTION
**Pull request by Benjozork**
Successor to #7586 

## Summary of Changes

This patch ports the PFD, EWD and ATSU to use `msfssdk` from npm, and removes our fork of the framework.

## Testing instructions

- Ensure all PFD indications function correctly
- Ensure all EWD indications function correctly
- Ensure all ATSU indications function correctly

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by tracernz — APPROVED** [review]

**Comment by beheh** [comment]
It was noted on Discord that the upstream version still does not support the `tspan` element in SVG which will impact NDv2.

File: .gitignore [hunk 1]
```diff
@@ -84,7 +82,6 @@
 
 # avoid issues when switching branches during transistion from old to new structure
 /src/
-/msfs-avionics-mirror/
```
Comment (tracernz) [inline .gitignore:87]: Can I suggest leaving all these gitignores for a few weeks at least to facilitate moving between and rebasing branches?

File: package.json [hunk 2]
```diff
@@ -164,7 +164,7 @@
     "immer": "^9.0.3",
     "lodash": "^4.17.20",
     "msfs-geo": "^0.1.0-alpha3",
-    "msfssdk": "file:fbw-common/msfs-avionics-mirror/src/sdk/build/msfssdk-0.1.0.tgz",
+    "@microsoft/msfs-sdk": "^0.4.0-dev4",
```
Comment (tracernz) [inline package.json:167]:
```suggestion
    "@microsoft/msfs-sdk": "^0.4.0",
```

**Comment by 2hwk** [comment]
QA Report
Testing:
- [x] Ensure all PFD indications function correctly
- [x] Ensure all EWD indications function correctly
- [x] Ensure all ATSU indications function correctly

Negatives:
- EFB not working... Will check if its just this local build, or an issue with this PR.
image

Media from Testing:

PFD/EWD:

image
image
image
image
image
image
image


ATSU:
image
image
image
image
image
image
image
image
image

Result:
- Not Passed

Conclusion:
Will check if EFB issue is this PR or a bad build

**Comment by 2hwk** [comment]
QA Report
Testing:
- [ ] Ensure all EFB functions work correctly
Not Tested: 
- [ ] Ensure all PFD indications function correctly 
- [ ] Ensure all EWD indications function correctly
- [ ] Ensure all ATSU indications function correctly

Negatives:
- Navigraph charts are broken

image

Media from Testing:
Result:
- Not Passed

Conclusion:
EFB is broken in this PR.

**Comment by 2hwk** [comment]
QA Report
Testing:
- [x] Ensure all PFD indications function correctly
- [x] Ensure all EWD indications function correctly
- [x] Ensure all ATSU indications function correctly
- [x] Ensure all EFB indications function correctly

Negatives:
- N/A (Se;f=Test broken, but this is a dev version issue, not related to this PR).

Result:
- Passed

Conclusion:
- LGTM

**Comment by 2hwk** [comment]
Fix Lint

**Pull request merged**